### PR TITLE
Format idea?

### PR DESCRIPTION
**Pull request by KrisNey-MSFT**

### CODE REVIEW

**Comment by mmiele** [comment]
This is the [repo structure](https://microsoft-my.sharepoint.com/:f:/p/v-mimiele/Eq-lS6-uN_1ImyKQwZYbEmcBJf0Ylo18hCmEd71AC2G6IQ?e=ImCvRG). It contains a lot of README files and seems difficult to navigate. I am concerned that many of these files will remain empty. Maybe we should flatten the structure to a certain extent.

**Comment by chrispsommers** [comment]
@mmiele The only "maintenance" is adding entries to the bottom-most folders when a new doc is created, providing easy finding and explanations. All the others are "done."

File: documentation/README.md [hunk 1]
```diff
@@ -19,6 +19,8 @@ All DASH devices shall conform to the following design specifications and compli
 | gNMI Northbound API                  | [Parent Folder](gnmi/README.md) \| [ Design](gnmi/design/README.md) \| [Compliance Requirements](gnmi/requirements/README.md)|
 | SAI Southbound API                   | [Parent Folder](sai/README.md) \| [ Design](sai/design/README.md) \| [Compliance Requirements](sai/requirements/README.md)|
 
+| General Architecture and Requirements | [Link to ReadMe w/Design & Compliance folders](general/README.md)
```
Comment (chrispsommers) [inline documentation/README.md:22]: This is already the first table row "design" link.

**Comment by KrisNey-MSFT** [comment]
Format agreed upon per Community Meeting - let's go with this structure.

**Pull request closed without merging**